client/state/action-types.js

```diff
@@ -36,6 +36,10 @@ export const ANALYTICS_STAT_BUMP = 'ANALYTICS_STAT_BUMP';
 export const AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST = 'AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST';
 export const AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE = 'AUTOMATED_TRANSFER_ELIGIBILITY_UPDATE';
 export const AUTOMATED_TRANSFER_STATUS_SET = 'AUTOMATED_TRANSFER_STATUS_SET';
+export const BILLING_DATA_RECEIVE = 'BILLING_DATA_RECEIVE';
+export const BILLING_DATA_REQUEST = 'BILLING_DATA_REQUEST';
+export const BILLING_DATA_REQUEST_FAILURE = 'BILLING_DATA_REQUEST_FAILURE';
+export const BILLING_DATA_REQUEST_SUCCESS = 'BILLING_DATA_REQUEST_SUCCESS';
```

Comment:
Actually, in my initial version of the code, I was using `BILLING_HISTORY` instead of `BILLING_DATA`. However, when I started implementing into components, I found out that it doesn't make much sense, because `history` implies we'll be using it only for past transactions; while on our case we're also using it for upcoming transactions. That's why I kept it `DATA`, however I'm also open to naming. 

Some other ways we could name it instead of `BILLING DATA` could be:

* `BILLING_ITEMS`
* `BILLING_ENTRIES`
* `BILLING_TRANSACTIONS`

@gwwar do you prefer any of those? If you do, I'd be happy to update the PR with that. I think `BILLING_TRANSACTIONS` is the best candidate, because it fits with both terms - `past` and `upcoming`.